infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -121,13 +121,18 @@ private int invalidateSmallestTxVersion() {
 
         int numEntries = 0;
         while (firstAddress() == firstEntry.txVersion) {
-            log.debug("evict items " + numEntries + " with address " + firstAddress());
+            if (log.isTraceEnabled()) {
+                log.trace("evict items " + numEntries + " with address " + firstAddress());
```

Comment:
can use log.trace(format, args) to avoid concatenation

Reply:
Done.